extension/src/popup/helpers/sorobanSwap.ts

```diff
@@ -0,0 +1,224 @@
+/* eslint-disable */
+
+import { xdr } from "stellar-sdk";
+import {
+  Router,
+  Token,
+  CurrencyAmount,
+  TradeType,
+  Networks,
+  Protocols,
+} from "soroswap-router-sdk";
```

Comment:
all the soroswap-router-sdk logic is in this file. I'm going to be adding some documentation to this so hopefully devs in the ecosystem can just grab these pieces of code and use them in their own wallets